build/crunchy-postgres-exporter/Dockerfile

```diff
@@ -1,32 +1,27 @@
-ARG BASEOS
-ARG BASEVER
-ARG PREFIX
-FROM ${PREFIX}/pgo-base:${BASEOS}-${BASEVER}
+FROM registry.access.redhat.com/ubi8/ubi-minimal
 
-ARG BASEOS
 ARG PGVERSION
-ARG PACKAGER
 
-LABEL name="crunchy-postgres-exporter" \
-	summary="Metrics exporter for PostgreSQL" \
-	description="When run with the crunchy-postgres family of containers, crunchy-postgres-exporter reads the PostgreSQL data directory and has a SQL interface to a database to allow for metrics collection." \
-	io.k8s.description="Crunchy PostgreSQL Exporter" \
-	io.k8s.display-name="Crunchy PostgreSQL Exporter" \
-	io.openshift.tags="postgresql,postgres,monitoring,database,crunchy"
```

Comment:
The RHCP preflight scans require these LABEL sections to be defined otherwise they will fail and we will be unable to publish any of these

Reply:
All the labels have been moved to the Makefile, so they should be present on the image after it gets built.

Reply:
📝 These are the [required labels](https://github.com/redhat-openshift-ecosystem/openshift-preflight/blob/7d2d41a2c4c111991d918213d0d24faebb27a9e3/internal/policy/container/has_required_labels.go#L15) that I know about.